src/google/TreeMap.js

```diff
@@ -0,0 +1,158 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "../common/HTMLWidget", "goog!visualization,1,packages:[treemap]"], factory);
+    } else {
+        root.TreeMap = factory(root.d3, root.common_HTMLWidget);
+    }
+}(this, function (d3, HTMLWidget) {
```

Comment:
@GordonSmith because Timeline and TreeMap each require their specific google packages ("goog!visualization,1,packages:[treemap]") Matt said it would be best to derive these from HTMLWidget.  I deferred to his experience, as I do to yours....  If there is a better way to do this, please explain.